addon/components/async-button.js

```diff
@@ -12,10 +12,17 @@ export default Ember.Component.extend({
   reset: false,
   classNames: ['async-button'],
   classNameBindings: ['textState'],
-  attributeBindings: ['disabled', 'type', '_href:href'],
+  attributeBindings: ['disableWhen', 'disabled', 'type', '_href:href', 'tabindex'],
 
   type: 'submit',
-  disabled: Ember.computed.equal('textState','pending'),
+  disabled: Ember.computed('textState', 'disableWhen', function() {
+      var textState = this.get('textState');
```

Comment:
quick question -it blows up when we transpile for tests (using let instead of var) -what deps should we update to enable tdd feedback w/ this vNext feature?